improver_tests/acceptance/SHA256SUMS

```diff
@@ -503,6 +503,7 @@ ae85179605e8c5bb8d23e8d5c74c78ee68ab326a71b399d247ee6a320a29e281  ./generate-rea
 b908105fe64ef8414812b48222ce0c593d858ff2aa8ed6253e122787a9e567d4  ./generate-realizations/percentiles_reordering/kgo.nc
 11dd99cd73abf34d3e88bcb0b892ece9e1f81624e6415ec151e608b97ec55815  ./generate-realizations/percentiles_reordering/multiple_percentiles_wind_cube.nc
 93070cf59f98b2f87e92ca4fbbebd8a2189f50610535f18d0da0711af8fac664  ./generate-realizations/percentiles_reordering/raw_forecast.nc
+35329bbbbc5c3859b538074092951b638f3267943e69b2d07d18f2fc0ae40955  ./generate-realizations/percentiles_reordering/tie_break_with_realization_kgo.nc
```

Comment:
I notice that this file is identical to the file produced when using the random tie break method. I think this is because a wind speed cube is being used, so there aren't any ties. It might be helpful if the files produced by the random and realization tie break methods were different, so perhaps we could use a precipitation forecast as the raw forecast to demonstrate the behaviour better.

Reply:
I've updated the inputs to use precipitation accumulation data and hence I've created non-identical KGOs. These are plotted [here](https://github.com/metoppv/improver_test_data/pull/48#issuecomment-2405205241).